blocks/ai-text/src/app/generate-text/text-preview.tsx

```diff
@@ -109,28 +114,46 @@ export const TextPreview = ({
           paddingX: 3.75,
         })}
       >
-        <Box display="flex" gap={1}>
+        <Box display="flex" gap={1} flexWrap="wrap">
           <Button
             size="small"
             onClick={onConfirm}
             sx={{
-              gap: 1,
               borderRadius: 1,
               fontSize: 14,
               fontWeight: 500,
               lineHeight: "18px",
             }}
           >
             Insert Into Page
+            <CheckIcon sx={{ fontSize: "inherit", ml: 1.25 }} />
           </Button>
 
           <Button
             variant="tertiary"
             size="small"
-            onClick={onDiscard}
+            onClick={onRegenerate}
             sx={{ fontSize: 14 }}
           >
-            Discard this
+            Regenerate output
+            <ArrowRotateLeftIcon sx={{ fontSize: "Inherit", ml: 1.25 }} />
+          </Button>
+
+          <Button
+            variant="tertiary_quiet"
+            size="small"
+            onClick={onDiscard}
+            sx={({ palette }) => ({
+              fontSize: 14,
+              color: palette.gray[50],
+              background: "transparent !important",
```

Comment:
Is there a way to easily achieve this using specificity? It's my view that in general it's worth trying to avoid introducing `!important` where possible.

Reply:
Sure, I did it initially so it would override the hovered background style value as well, but I don't mind changing it. a67532916fa7be89d0a7b059305a4dc23497b497